exchanges/account/account.go

```diff
@@ -103,7 +104,7 @@ func GetHoldings(exch string, creds *Credentials, assetType asset.Item) (Holding
 		return Holdings{}, fmt.Errorf("%s %s %s %w %w", exch, creds, assetType, errNoCredentialBalances, ErrExchangeHoldingsNotFound)
```

Comment:
If you could also update `GetHoldings` to actually return the `UpdatedAt` value, it would be handy to receive